test/e2e/handler/default_bridged_network_with_nmpolicy_test.go

```diff
@@ -0,0 +1,132 @@
+package handler
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstate "github.com/nmstate/kubernetes-nmstate/api/shared"
+)
+
+var _ = Describe("NodeNetworkConfigurationPolicy default bridged network with nmpolicy", func() {
```

Comment:
I see too much duplication here, maybe it's better to just parameterize the test with desiredState so we use capture and non capture version, also in the future we will add a non DHCP default bridge here on the `default_bridged_network_test.go`

Reply:
As we discussed offline, the test avoid duplication as much as possible and follows the principals of the other handler e2e tests.